packages/@okta/vuepress-site/docs/reference/api/push-providers/index.md

```diff
@@ -0,0 +1,625 @@
+| `tokenSingingKey`    | String          | APNs private authentication token signing key      | `APNS`                                   |
+
+#### Push Provider object example (APNS provider type)
+
+```json
+{
+  "providerType": "APNS",
+  "name": "Example Push Provider 1",
+  "configuration": {
+    "keyId": "KEY_ID",
+    "teamId": "TEAM_ID",
+    "tokenSigningKey": "-----BEGIN PRIVATE KEY-----\nPRIVATE_KEY\n-----END PRIVATE KEY-----\n",
+    "fileName": "fileName.p8"
+  }
+}
+```
+
+#### Push Provider object example (FCM provider type)
+
+```json
+{
+  "providerType": "FCM",
+  "name": "Config 1",
+  "configuration": {
+    "serviceAccountJson": {
+        "type": "service_account",
+        "project_id": "PROJECT_ID",
+        "private_key_id": "KEY_ID",
+        "private_key": "-----BEGIN PRIVATE KEY-----\nPRIVATE_KEY\n-----END PRIVATE KEY-----\n",
+        "client_email": "SERVICE_ACCOUNT_EMAIL",
+        "client_id": "CLIENT_ID",
+        "auth_uri": "https://accounts.google.com/o/oauth2/auth",
+        "token_uri": "https://accounts.google.com/o/oauth2/token",
+        "auth_provider_x509_cert_url": "https://www.googleapis.com/oauth2/v1/certs",
+        "client_x509_cert_url": "https://www.googleapis.com/robot/v1/metadata/x509/SERVICE_ACCOUNT_EMAIL"
+    },
+    "fileName": "fileName.json"
+  }
+}
+```
+
+### Push Provider response object
+
+#### Push Provider response properties
+
+The Push Provider response object has the following properties:
+
+| Property          | Type                                   | Description                                        |
+| ----------------- | -------------------------------------- | -------------------------------------------------- |
+| `_links`          | [Link](#link-object)                   | Allowed operations for the Push Provider           |
+| `configuration`   | [Configuration response](#configuration-response-object) | Object that contains providerType specific configurations |
+| `id`              | String                                 | Unique key for the Push Provider                   |
+| `lastUpdatedDate` | String (ISO-8601)                      | Timestamp when the Push Provider was last modified |
+| `name`            | String                                 | Display name of the Push Provider                  |
+| `providerType`    | String                                 | Provider type for the Push Provider. One of `APNS`, `FCM` |
+
+#### Link object
+
+For a Push Provider result, the `_links` contains a full set of operations available for that Push Provider using the [JSON Hypertext Application Language](https://tools.ietf.org/html/draft-kelly-json-hal-06) specification. `hints` provides information on allowed HTTP verbs for the `href`. The Links object is read-only. See [Web Linking](https://tools.ietf.org/html/rfc8288)) for more information on link relations.
```

Comment:
I cut out a fair bit of text here. The opening phrase "For a Push Provider result" already captures a specific PP.

```suggestion
For a Push Provider result, the `_links` property is read-only and contains a full set of available operations using the [JSON Hypertext Application Language](https://tools.ietf.org/html/draft-kelly-json-hal-06) specification. The `hints` property provides information on allowed HTTP verbs for the `href` property. See [Web Linking](https://tools.ietf.org/html/rfc8288) for more information on link relations.
```